card/lib/card/migration.rb

```diff
@@ -29,11 +29,11 @@ def find_unused_name base_name
     end
 
     def paths mig_type=type
-      Cardio.paths["db/migrate#{schema_suffix mig_type}"].to_a
+      Card.paths["db/migrate#{schema_suffix type}"].to_a
```

Comment:
That looks like a bug.  The merge of the last refactoring of the migrations support didn't go completely right.